de.json

```diff
@@ -151,9 +151,9 @@
         "World.Actions.HostingWarning" : "Sie sind Host dieser Welt. Alle Nutzer werden getrennt, wenn Sie die Welt schließen.",
 
         "World.Actions.GetWorldOrb" : "Welt-Orb erzeugen",
-        "World.Actions.CopyWorldURL" : "World-URL in Zwischenablage kopieren",
+        "World.Actions.CopyWorldURL" : "Welt-URL in Zwischenablage kopieren",
```

Comment:
Mine is "World-URL kopieren"
(gonna commit the change to "Welt-URL kopieren" so we use the German word haha)